cmd/root.go

```diff
@@ -93,13 +96,20 @@ func initConfig() {
 	viper.MergeInConfig()
 
 	if isConfigExtends() {
-		filename := filepath.Base(getExtendsPath())
-		extension := filepath.Ext(getExtendsPath())
-		name := filename[0 : len(filename)-len(extension)]
-		viper.SetConfigName(name)
-		viper.AddConfigPath(filepath.Dir(getExtendsPath()))
-		err := viper.MergeInConfig()
-		check(err)
+		if isValidUrl(getExtendsPath()) {
+			uri := getExtendsPath()
+			baseRemoteConfig := fetchRemoteExtends(uri)
+			err := viper.MergeConfig(baseRemoteConfig)
```

Comment:
I didn't call `viper.SetConfigName` prior to this. I'm not sure if that's necessary or not.